src/ac_copilot_trainer/modules/coaching_overlay.lua

```
@@ -117,11 +117,43 @@ end
+local _ovDiagLogged = false
 function M.drawApproachPanel(approachData)
-  if not ui or type(vec2) ~= "function" then
+  if not _ovDiagLogged and ac and type(ac.log) == "function" then
+    _ovDiagLogged = true
+    local function tt(t, k)
+      if type(t) ~= "table" then return "?" end
+      local v = t[k]
+      if v == nil then return "nil" end
+      return type(v)
+    end
+    local szStr = "err"
```

Luacheck (1.2.0)

[warning] value assigned to variable 'szStr' is unused

(W311)